### ISSUE TITLE
Performance Issue - Slow Dragging and Dropping Files/Folders

### ISSUE THREAD

**Issue by 6StringSam** [mugiwara85/CodeblockCustomizer#111]
I'm experiencing slowness dragging files and folders. The problem persists with all other plug-in disabled.  Replacing the style.css with an empty file fixed the slow file/folder dragging issue.  The vault with the issue has 6,000 notes.  Added a video of the issue.  https://github.com/user-attachments/assets/8aa0544c-2ecf-4ce1-9fdd-01758ebf54a1

Obsidian 1.7.3
Codeblock Customizer: 1.2.8

The issue is similar to this issue from a while back.  https://github.com/mugiwara85/CodeblockCustomizer/issues/54

Tested in a test vault (100 files) and it is not an issue.  Not sure how the size of the vault could be an issue.  Tied on my laptop and it is not an issue....

Next, I'm going to un-install and reinstall Obsidian to see if that has an impact.

**Comment by mugiwara85** [comment]
No need to reinstall Obsidian. Which Theme are you using? Also, when you tried in a test Vault with less files, where you using the same theme?

**Comment by 6StringSam** [comment]
I am using the minimal theme but set it back to default as a troubleshooting step.  

I downloaded the latest and version of Obsidian 1.6.7 and installed it.  (Didn't un-install first, tried the lazy path first.) That fixed the issue!  Maybe some underlining Electron dependency...

**Comment by mugiwara85** [comment]
Wait. You said in your first message that you are using Obsidian 1.7.3. But I just saw that the current version is 1.6.7. So, you were using an older version of Obsidian?

It is working now correctly, right?

This behavior is definitely a CSS problem. It is also possible that the problem was in Obsidian CSS, or in a Themes CSS, or that there was some CSS which conflicted with the plugins CSS, or the plugins CSS conflicted with one of them. Anyway, it is hard to debug, that's why I asked you to open a new issue, since I already fixed the other one. That shouldn't be a problem anymore.
